cli/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -1531,64 +1531,77 @@ pub(super) mod test {
 
-            # The `hook.on-activate` script is run by the *bash* shell immediately upon
-            # activating an environment, and will not be invoked if Flox detects that the
-            # environment has previously been activated. Variables set by the script will
-            # be inherited by `[profile]` scripts defined below. Note that any stdout
-            # generated by the script will be redirected to stderr.
+
+            ## Activation Hook ----------------------------------------------------------
+            ##  ... run by _bash_ shell when you run 'flox activate'.
+            ## --------------------------------------------------------------------------
             [hook]
             # on-activate = '''
-            #   # Set variables, create files and directories
-            #   venv_dir="$(mktemp -d)"
-            #   export venv_dir
-            #
-            #   # Perform initialization steps, e.g. create a python venv
-            #   python -m venv "$venv_dir"
+            #   # -> Set variables, create files and directories
+            #   # -> Perform initialization steps, e.g. create a python venv
             # '''
 
-            # Scripts defined in the `[profile]` section are *sourced* by *your shell* and
-            # inherit environment variables set in the `[vars]` section and by `[hook]` scripts.
-            # The `profile.common` script is sourced by all shells and special care should be
-            # taken to ensure compatibility with all shells, after which exactly one of
-            # `profile.{bash,fish,tcsh,zsh}` is sourced by the corresponding shell.
+
+            ## Profile script -----------------------------------------------------------
+            ## ... sourced by _your shell_ when you run 'flox activate'.
+            ## --------------------------------------------------------------------------
             [profile]
             # common = '''
-            #   echo "it's gettin' flox in here"
+            #   gum style \
+            #   --foreground 212 --border-foreground 212 --border double \
+            #   --align center --width 50 --margin "1 2" --padding "2 4" \
+            #     $INTRO_MESSAGE
             # '''
+            ## Shell specific profiles go here:
+            # bash = ...
+            # zsh  = ...
+            # fish = ...
 
-            # The `[services]` section of the manifest allows you to define services.
-            # Services defined here use the packages provided by the `[install]` section
-            # and any variables you've defined in the `[vars]` section or `hook.on-activate` script.
+
+            ## Services -----------------------------------------------------------------
+            ##  $ flox services start   <- Starts all services
+            ##  $ flox services status  <- Status of running services
+            ## --------------------------------------------------------------------------
```

Comment:
@zmitchell @dcarley there's already `flox services start`, but maybe we should include `activate -s` here?

`##  $ flox activate -s  <- Starts all services when entering environment`